subworkflows/local/utils_nfcore_seqinspector_pipeline/main.nf

```diff
@@ -75,29 +75,33 @@ workflow PIPELINE_INITIALISATION {
     //
     // Custom validation for pipeline parameters
     //
-    validateInputParameters()
+    validateInputParameters() // Runs additional validation that is not done by $projectDir/nextflow_schema.json
 
     //
     // Create channel from input file provided through params.input
     //
     Channel
-        .fromSamplesheet("input")
+        .fromSamplesheet("input") // Validates samplesheet against $projectDir/assets/schema_input.json. Path to validation schema is defined by $projectDir/nextflow_schema.json
         .map {
             meta, fastq_1, fastq_2 ->
+                def id_string = "${meta.sample}_${meta.group ?: "ungrouped"}_${meta.lane}"
```

Comment:
```suggestion
                def id_string = "${meta.sample}_${meta.project ?: "ungrouped"}_${meta.lane}"
```
This should be `project`, right? Or should we stick to `group` everywhere? (in `schema_input.json` it's `project` now).

Reply:
Using `group` instead of `project` might make it more intuitive to apply for non-facility pipeline users. We could clarify it in the documentation either way.

Reply:
I think we should use group. I guess we need an argument so that NGI can still call the reports proejct-reports?

Reply:
@Aratz I agree now, it seems project has been used everywhere else. So either we change everything to group or we stick with project.